cosmos/profiles/postgres/user_pass.py

```diff
@@ -40,7 +40,7 @@ class PostgresUserPasswordProfileMapping(BaseProfileMapping):
     def profile(self) -> dict[str, Any | None]:
         "Gets profile. The password is stored in an environment variable."
         profile = {
-            "port": 5432,
```

Comment:
Hi @farovictor , thanks for your contribution!

There was a recent change which makes the connection information override the hard-coded default value, and we had a test for it: #511 

Could you confirm if this is not working and add a change to your PR if it is solving a different scenario?

Reply:
It does work. My bad, I was using a non-up-to-date version. Closing this PR.